src/content/en/agencies/js/directory.js

```diff
@@ -0,0 +1,656 @@
+/**
+ * @fileoverview Creates web directory of agencies based on JSON data
```

Comment:
needs our copyright header

Reply:
Not sure if I'm familiar, is there an example of this in the repo?

Reply:
@ebidel made the changes you suggested along with running it through the linter.  One thing I can't fix however is the use of let/const since we have to use ES5 within devsite.

Any documentation/example of this copyright header?